### ISSUE TITLE
Use system notification to accept receiving requests

### ISSUE THREAD

**Issue by QWERT-commits** [nearby-sharing/android#47]
Currently, receiving requests can only be accepted if you open the app and click on the "Accept" option.
If the app stays in the background and pop up a notification that allow users to accept/reject files sent by the PC, it would be more similar to the nearby sharing experience between PC.

**Comment by ShortDevelopment** [comment]
Depends on #41

**Comment by ShortDevelopment** [comment]
Duplicate of #41
